OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -283,21 +285,23 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		abstract class MovePart : Activity
+		// NOTE: This class is uncancellable until it completes its intended move...
+		// If we could solve this, we could enable path smoothing!
+		public abstract class MovePart : Activity
```

Comment:
These just feel like implementation details that shouldn't be publicly exposed to me. I'll let somebody else weight in.